tests/testthat/test-mlaps.R

```diff
@@ -0,0 +1,171 @@
+# Helper function to create dummy lab data needed for mLAPS unit tests
+dummy_lab <- function(id, omop, value, unit, mintime){
+  res <- data.table(
+    genc_id = rep(id, length(value)),
+    test_type_mapped_omop = omop,
+    result_value = value,
+    result_unit = rep(unit, length(value)),
+    collection_date_time =  format(as.POSIXct(mintime, tz = "UTC") + sample(0:(24*60*60 - 1), size=length(value), replace = TRUE),"%Y-%m-%d %H:%M")
+  )
+  return(res)
+}
+
+# Helper function to create dummy admdad data needed for mLAPS unit tests
+dummy_admdad <- function(id, admtime){
```

Comment:
Note that branch `14_plotting_funcs` has a `dummy_ipadmdad` function, which is a bit more general (see here: https://github.com/GEMINI-Medicine/Rgemini/blob/14_plotting_funcs/R/dummy_data.R). You could copy and paste the code from branch 14 into `dummy_data.R` and then use that - it might be useful to have that function on develop anyway (although it hasn't been reviewed...). Anyway, up to you - it it's too much trouble to switch it over at this point, feel free to leave it as is.

Reply:
I'd like to wait until that branch is merged and then make the change (perhaps we can create an issue for this) to avoid potential merge conflicts?

Reply:
Sounds good! I'll create a new issue and add the `dummy_ipadmdad()` function to a new branch (don't want to merge the rest of the code on branch 14 yet...). I'll mention that the mlaps test code should be updated accordingly once the `dummy_ipadmdad()` function has been reviewed.